org.hl7.fhir.publisher.core/src/main/java/org/hl7/fhir/igtools/publisher/PublisherIGLoader.java

```diff
@@ -3074,11 +3135,26 @@ private void loadBundles(FetchedFile igf) throws Exception {
     for (String be : pf.bundles) {
       loadBundle(be, igf, "listed as a bundle");
     }
+    for (String ar : pf.additionalResourceFiles) {
+      loadSearchBundle(getSearchParamsFileName(ar), igf, "additional Resource Search Bundle", getSearchParamsResourceName(ar));
+    }
   }
 
   private boolean loadBundle(String name, FetchedFile igf, String cause) throws Exception {
```

Comment:
Neither of the boolean results of these  loadBundleMethods actually get read anywhere.

Reply:
no. and once upon it mattered, but not now